crowbar_framework/config/routes.rb

```diff
@@ -118,6 +118,18 @@
       end
     end
 
+    # service restart management
+    resource :restart_management,
```

Comment:
Please use the new API v2 namespace.

Reply:
You gotta be more explicit than that :D

Im guessing you mean setting the routes under the v2 namespace like `/utils/2.0/restart_management` ? Or a different one than that?

We dont have the other util routes under `1.0` which is bad :(

Reply:
Or you mean to put everything under the `/api/` namespace?

Reply:
Somewhere below the following lines would be nice.
```
  namespace :api,
    constraints: ApiConstraint.new(2.0) do
```